tools/app/app/cards/[key]/edit/page.tsx

```diff
@@ -106,121 +62,117 @@ export default function Page({ params }: { params: { key: string } }) {
     }
   }
 
-  const getPreview = useCallback(() => {
-    if (!card) return null
+  const preview = watch()
+
+  const previewCard = useMemo(() => {
+    const { __content__, __title__, ...metadata } = preview
     return {
       ...card!,
-      content: getValues('__content__'),
       metadata: {
         ...card!.metadata!,
-        summary: getValues('__title__'),
+        summary: __title__,
+        ...metadata,
       },
+      content: __content__,
     }
-  }, [card, getValues])
+  }, [preview, card])
 
-  if (!isReady) return <CircularProgress />
+  useEffect(() => {
```

Comment:
This seems like a development-time debug stuff that should be removed?

Reply:
Yes, and I removed it